tc/sgx/trusted_worker_manager/enclave/work_order_preprocessed_keys.h

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#pragma once
+
+#include <string>
+
+#include "types.h"
+#include "work_order_data.h"
+#include "enclave_data.h"
+
+class WorkOrderPreProcessedKeys {
+public:
+    WorkOrderPreProcessedKeys() {
+        sym_key = {};
+        work_order_session_key = {};
+        signing_key = {};
+        verification_key = {};
+        verification_key_signature = {};
+        signature = {};
+    }
+
+    tcf_err_t ProcessPreProcessedWorkOrderKeys(
+        std::string work_order_keys_data_json, EnclaveData* enclave_data);
+
+    ByteArray DecryptKey(const ByteArray& encryption_key,
+        const ByteArray& key_to_decrypt);
+
```

Comment:
Please describe each members of the class

Reply:
Done